frontend/src/component/changeRequest/ChangeRequest/Changes/Change/SegmentChange.tsx

```diff
@@ -20,61 +21,67 @@ export const SegmentChange: FC<ISegmentChangeProps> = ({
     onNavigate,
     actions,
     changeRequestState,
-}) => (
-    <Card
-        elevation={0}
-        sx={(theme) => ({
-            marginTop: theme.spacing(2),
-            marginBottom: theme.spacing(2),
-            overflow: 'hidden',
-        })}
-    >
-        <Box
+}) => {
+    const { segment } = useSegment(segmentChange.payload.id);
```

Comment:
The main change here is using the segment from the change to get the segment name.